app/src/androidTest/assets/pages/sampleText.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+    <title>Sample_Text</title>
```

Comment:
Just out of curiosity... could we use an exisiting site? like: https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/androidTest/assets/pages/lorem-ipsum.html

Reply:
That's what I've did, but the text selection proved be very unreliable, in most cases it selected the blank spaces or the wrong words.

Reply:
oh I see...

Reply:
what about this one? it is pretty similar... https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/androidTest/assets/pages/generic1.html#L3

Reply:
Overlooked this resource, sorry for that. :blush:

Checked locally and again the long click on the desired text was imprecise.

Tried selecting the following:
❌ "Page" - actually selected content
✔️ "content" - worked properly
❌ "Page content:" - actually selected content
❌ "Page content: 1" - actually selected content

Based on this, re-ran the test 10x on Firebase and successfully passed using "content"